docs/concepts/what-is-an-activity-task.md

```diff
@@ -2,9 +2,20 @@
 id: what-is-an-activity-task
 title: What is an Activity Task?
 sidebar_label: Activity Task
-description: An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+description: An Activity Task contains the context needed to make an Activity Task Execution.
 tags:
   - explanation
 ---
 
-An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+An Activity Task contains the context needed to proceed with an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution).
+From the API, users see an Activity being executed and assume that it runs once.
+
+Activity Tasks contain Events that are needed to execute the Activity. These events are found within the Activity Task's `Context`.
```

Comment:
I would change this to - 
Activity Tasks largely represent the Activity Task Scheduled Event which contains the data needed to execute an Activity Function.
If Heartbeat data is being passed, an Activity Task will also contain the latest Heartbeat details for example.

Something like that maybe